Ginger/GingerConsole/Program.cs

```diff
@@ -59,10 +64,8 @@ static void Main(string[] args)
                 Keepalive = false;                      
             };
 
-            Console.WriteLine();
-            Console.WriteLine("---------------------------------------------------");
-            Console.WriteLine("-            Press CTRL+C to exit                 -");
-            Console.WriteLine("---------------------------------------------------");
+
+            Amdocs.Ginger.CoreNET.log4netLib.GingerLog.InitLog4Net();
```

Comment:
1- need to move the initLog4Net to begining of Main()
2- need to use Reporter.ToLog() instead Console.Writeline() so all will be documented in Log file as well for debug if needed

Reply:
Done